parser.y

```diff
@@ -5789,6 +5788,37 @@ TableNameList:
 		$$ = append($1.([]*ast.TableName), $3.(*ast.TableName))
 	}
 
+TableNameOptWild:
```

Comment:
Could this be replaced as `TableName OptWild`? An extra rule doesn't seem necessary.

Reply:
I think we need this to avoid shift/reduce conflicts.

Reply:
> Could this be replaced as `TableName OptWild`? An extra rule doesn't seem necessary.

I think we need this to avoid shift/reduce conflicts.

Reply:
Okay.